pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/extensions/scheduler/TransferShedder.java

```diff
@@ -500,6 +501,13 @@ public Set<UnloadDecision> findBundlesForUnloading(LoadManagerContext context,
                         }
                         continue;
                     }
+                    final String namespaceName = NamespaceBundle.getBundleNamespace(bundle);
+                    if (sheddingExcludedNamespaces.contains(namespaceName)) {
+                        if (debugMode) {
+                            log.info("Skipping load shedding for namespace {}", namespaceName);
```

Comment:
can we prefix this string?
log.info(String.format(CANNOT_UNLOAD_BUNDLE_MSG, " Bundle namespace has been found in sheddingExcludedNamespaces", bundle)